LoopKitUI/ViewModels/TherapySettingsViewModel.swift

```diff
@@ -13,28 +13,42 @@ import SwiftUI
 
 public class TherapySettingsViewModel: ObservableObject {
     public typealias SaveCompletion = (TherapySetting, TherapySettings) -> Void
+   
+    /// This method type describes a way to "precheck" before saving max temp basal.  The host app is going to supply a closure for this, and based on the
+    /// response, either proceed with saving max temp, or display an error.
+    ///
+    /// - Parameters:
+    ///   - unitsPerHour: The temporary basal rate proposed to validate, in international units per hour
+    ///   - completion: A closure called after the command is complete
+    ///   - error: An optional error describing why the command failed
+    public typealias ValidateMaxTempBasal = (_ unitsPerHour: Double, _ completion: @escaping (_ error: Error?) -> Void) -> Void
```

Comment:
This is a hard one to name... keep trying to find something that covers the intent better. Maybe maxTempBasalSavePreflight?  The current name reads like we're trying make sure the passed max temp is valid. But yeah, it's tough.

Reply:
It's ugly to be sure... and probably a code smell.  I'm not a huge fan of the term "Preflight" (it's unusual) but I would like to get this PR in.